### PR TITLE
GSC UI Copy

### PR DESCRIPTION
**Pull request by cashd**
- Copy for GSC page
- invalidate cache after joining GSC
- useCandidates hook fetches knownDelegate info
- idle gsc history tooltip

### CODE REVIEW

**Review by DannyDelott — APPROVED** [review]

File: apps/elf-council-frontend/src/ui/gsc/GSCOverviewSection.tsx [hunk 1]
```diff
@@ -143,9 +143,14 @@ export function GSCOverviewSection(): ReactElement {
                       />
                     </Disclosure.Button>
                     <Disclosure.Panel className="flex max-w-fit flex-col gap-3 px-4 pt-4 pb-2 text-sm text-gray-500">
-                      <p>{t`Council is an on-chain decentralized governance system through which a community can manage a DAO. It gives the community total flexibility over how to distribute Voting Power and allows it to adapt its governance system to the continuously evolving needs of the DAO.`}</p>
-                      <p>{t`The system also includes the optional structure of a Governance Steering Council (GSC) with added governance powers and responsibilities, all to be decided upon by the community.`}</p>
-                      <p>{t`This flexibility is possible thanks to the use of Voting Vaults. Learn more in `}</p>
+                      <p>{t`Any delegate who has accumulated more delegated voting
+                      power than the current GSC Eligibility Threshold (110,000
```
Comment (hnguyen1179) [inline apps/elf-council-frontend/src/ui/gsc/GSCOverviewSection.tsx:147]: Q: Do we wanna use the `formattedThreshold` variable used above for this one here too?

File: apps/elf-council-frontend/src/ui/gsc/GSCOverviewSection.tsx [hunk 2]
```diff
@@ -187,9 +198,25 @@ export function GSCOverviewSection(): ReactElement {
...
-                      <p>{t`The system also includes the optional structure of a Governance Steering Council (GSC) with added governance powers and responsibilities, all to be decided upon by the community.`}</p>
-                      <p>{t`This flexibility is possible thanks to the use of Voting Vaults. Learn more in `}</p>
+                      <p>{t`Joining and leaving the GSC are individual on-chain transactions.`}</p>
+                      <ul className="list-disc">
+                        <li>
+                          <span className="font-bold">{t`Joining: `}</span>
+                          {t`If a delegate who isn’t part of the GSC
+                          acquires more delegated Voting Power than the GSC
+                          Eligibility Threshold (110,000 ELFI), they can join
+                          the GSC by using the “Join” button that will become
```
Comment (hnguyen1179) [inline apps/elf-council-frontend/src/ui/gsc/GSCOverviewSection.tsx:208]: Same with this

File: apps/elf-council-frontend/src/ui/gsc/GSCOverviewSection.tsx [hunk 3]
```diff
@@ -187,9 +198,25 @@ export function GSCOverviewSection(): ReactElement {
...
+                          enabled in the GSC’s Overview page.`}
+                        </li>
+                        <li>
+                          <span className="font-bold">{t`Leaving: `}</span>
+                          {t`If a GSC member falls below the GSC Eligibility
+                          Threshold (110,000 ELFI), anyone can execute the
+                          necessary on-chain transaction to formally remove them
```
Comment (hnguyen1179) [inline apps/elf-council-frontend/src/ui/gsc/GSCOverviewSection.tsx:215]: and this

**Review by hnguyen1179 — APPROVED** [review]
lgtm--just one question

**Pull request merged**